.envSample

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
This file should be renamed to `.env` on your local repo. It will contain the URL to your test and development databases. We do not need to merge this file, `.env` has been added to `.gitignore` already

Reply:
I already a .env in my local environment.
it is just sample as i have named to show whoever is cloning how to configure the database.